java/src/main/java/ai/rapids/cudf/RmmAllocationMode.java

```diff
@@ -32,4 +32,8 @@
    * Use arena suballocation strategy
```

Comment:
2019-2021 copyrights (if it remains modified)

Reply:
Reverted.